packages/smooth_app/lib/cards/product_cards/smooth_product_card_template.dart

```diff
@@ -0,0 +1,80 @@
+import 'package:flutter/material.dart';
+import 'package:smooth_app/cards/product_cards/smooth_product_card_found.dart';
+import 'package:smooth_app/generic_lib/design_constants.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_card.dart';
+import 'package:smooth_app/generic_lib/widgets/smooth_product_image_container.dart';
+import 'package:smooth_app/helpers/ui_helpers.dart';
+
+/// Empty template for a product card display.
+///
+/// Based on the "real" [SmoothProductCardFound].
+class SmoothProductCardTemplate extends StatelessWidget {
+  const SmoothProductCardTemplate();
+
+  @override
+  Widget build(BuildContext context) {
+    final Size screenSize = MediaQuery.of(context).size;
+    final ThemeData themeData = Theme.of(context);
+    final bool isDarkMode = themeData.colorScheme.brightness == Brightness.dark;
+    final Color itemColor = isDarkMode ? PRIMARY_GREY_COLOR : LIGHT_GREY_COLOR;
+    final Color backgroundColor = isDarkMode ? Colors.black : Colors.white;
+    final double iconSize = IconWidgetSizer.getIconSizeFromContext(context);
+    final Widget textWidget = Container(
+      width: screenSize.width * .4,
+      height: screenSize.width * .05,
+      color: itemColor,
+    );
+    // In the actual display, it's a 240x130 svg resized with iconSize
+    final Widget svgWidget = Container(
+      height: iconSize * .9,
+      width: 240 * iconSize / 130,
```

Comment:
I guess you took these values from the original card. We should store them in a const

Reply:
Fair enough, but I don't know how to do that cleanly. You see, the 240x130 ratio comes from the svg files themselves.
The general idea is to have a template that looks similar to a real product card, not to match it in every detail, especially for a widget that is transient.